proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+
+### Bylaws requirements
+- Communicated scope for the foundation and board 
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement 
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem. 
+- Project representation on the Board 
+    - Community Representation should sit on the Board directly. Projects could share this representation. 
+    - Community representation on the Board: 3 seats guaranteed for community representation 
+- Board must establish Cross Project Council focused on the specific needs of the member projects 
+    - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board 
+- Non-corporate member representation in budget/finance subcommittee 
+- Projects values should drive foundation focus and priority 
+- No "pay to play" in technical participation 
+- Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects 
+- Value satisfaction: regular feedback from projects 
+- Explicit acknowledgement and documention for the ability of projects to leave the foundation 
+- Foundation should document a clear process for trademark usage to enable community not-for-profit efforts
+- Communicated scope for the foundation and board
+  - Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement
+  - The Foundation staff, executive director, and board should follow a "servant leadership" style.
+  - Non-corporate member representation in budget / finance subcommittee
+  - Projects values should drive foundation focus and priority
+  - No "pay to play" in technical participation
+  - Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects
+  - Value satisfaction: regular feedback from projects
+- Executive Director required to work directly with projects to gather strategy and budget needs for upcoming roadmaps including but not limited to: 
+  - Brand management per Project
+  - Budget Allocation per Project
+  - Resource Allocation per Project
+
+### Communicated expectations
+
+##### Expectations of the Foundation
+- Protect Intellectual Property (IP) including trademarks 
+    - Enable the project communities to use the trademarks 
+    - Educate members, new and old, about proper use of the projects' IP 
+
+- Provide Legal Advice and Legal Best Practices 
+    - Provide legal resources and representation that support the needs of the project 
+    - Create a well-defined communications path for project legal needs 
```

Comment:
I would like to add `within a 3-6 month timeframe`.

Reply:
+1